### PR TITLE
Fixup config

### PR DESCRIPTION
**Pull request by darosior**
A small one on top of #12, to fixup the config (as managers are not anymore always part of the stakeholders).

Unfortunately i had to reintroduce a dev dep, see commit messages..

### CODE REVIEW

**Comment by darosior** [comment]
Rebased on #12 after #10 merge

**Comment by darosior** [comment]
CI is flaky because we pull deps from Github :sob:

**Comment by darosior** [comment]
Rebased after test integration in #12

File: src/database/interface.rs [hunk 1]
```diff
@@ -72,16 +73,46 @@ pub struct DbWallet {
...
     pub vault_descriptor: String,
     pub unvault_descriptor: String,
+    pub ourselves: config::OurSelves,
 }
 
 /// Get the database wallet. We only support single wallet, so this always return the first row.
 pub fn db_wallet(db_path: &PathBuf) -> Result<DbWallet, DatabaseError> {
     let rows = db_query(db_path, "SELECT * FROM wallets", NO_PARAMS, |row| {
+        let our_man_xpub_str = row.get::<_, Option<String>>(4)?;
+        let our_man_xpub = if let Some(ref xpub_str) = our_man_xpub_str {
```
Comment (edouardparis) [inline src/database/interface.rs:83]: nit,
I may prefer:
```rust
let our_man_xpub = our_man_xpub_str.map(|ref xpub_str| {
 DescriptorPublicKey::from_str(&xpub_str)
                    .map_err(|e| FromSqlError::Other(Box::new(e)))?
} )
```

**Comment by edouardparis** [comment]
ACK 12c712d7b1b85720203d763d20bed1d1a39c67bc

**Comment by darosior** [comment]
ACK 0769bd4

**Pull request merged**